pip/pip-375.md

```diff
@@ -0,0 +1,58 @@
+# PIP-375: Expose the Admin client configs: readTimeout, requestTimeout, connectionTimeout
+
+# Background knowledge
+
+- `pulsar-admin sinks create` will upload files to brokers, but if the file is large and the network is slow, it may cost a long time.
+- The default value of `requestTimeout` of Admin API is `60s`, after the fix https://github.com/apache/pulsar/pull/23128, it was changed to `300s`.
+
+# Motivation
+
+We'd better to expose the following configs to users:
```

Comment:
Why is it better?

Reply:
See https://github.com/apache/pulsar/pull/23222/files#diff-be84b11fa7ada15d3ef3a329b5ec3b0cd798c9d50d70d2d8c73e77714592d547R5

Reply:
It would be good to update this information in the PIP document.